### PR TITLE
adis1650x and adis1657x projects: Add scale reading to dummy example

### PR DESCRIPTION
**Pull request by rbolboac**
## Pull Request Description

Please replace this with a detailed description and motivation of the changes. 
You can tick the checkboxes below with an 'x' between square brackets or just check them after publishing the PR. 
If this PR contains a breaking change, list dependent PRs and try to push all related PRs at the same time.

## PR Type
- [ ] Bug fix (change that fixes an issue)
- [x] New feature (change that adds new functionality)
- [ ] Breaking change (has dependencies in other repos or will cause CI to fail)

## PR Checklist
- [x] I have followed the [Coding style guidelines](http://analogdevicesinc.github.io/no-OS/drivers_guide.html#coding-style)
- [x] I have performed a self-review of the changes
- [x] I have commented my code, at least hard-to-understand parts
- [x] I have build all projects affected by the changes in this PR
- [x] I have tested in hardware affected projects, at the relevant boards
- [x] I have signed off all commits from this PR
- [x] I have updated the documentation (wiki pages, ReadMe etc), if applies

### CODE REVIEW

File: projects/eval-adis1650x/src/examples/dummy/dummy_example.c [hunk 1]
```diff
@@ -96,14 +86,39 @@ int dummy_example_main()
...
+	ret = adis_get_anglvel_scale(adis1650x_desc, &anglvel_scale);
+	if (ret)
+		goto error_remove;
+
+	ret = adis_get_temp_scale(adis1650x_desc, &temp_scale);
+	if (ret)
+		goto error_remove;
+
+	float output_scale[] = {
+		1.0 * anglvel_scale.dividend/anglvel_scale.divisor,
```
Comment (amiclaus) [inline projects/eval-adis1650x/src/examples/dummy/dummy_example.c:113]: nit: spaces missing

File: projects/eval-adis1657x/src/examples/dummy/dummy_example.c [hunk 2]
```diff
@@ -97,13 +87,38 @@ int dummy_example_main()
...
+	ret = adis_get_anglvel_scale(adis1657x_desc, &anglvel_scale);
+	if (ret)
+		goto error_remove;
+
+	ret = adis_get_temp_scale(adis1657x_desc, &temp_scale);
+	if (ret)
+		goto error_remove;
+
+	float output_scale[] = {
+		1.0 * anglvel_scale.dividend/anglvel_scale.divisor,
```
Comment (amiclaus) [inline projects/eval-adis1657x/src/examples/dummy/dummy_example.c:113]: nit: spaces missing

**Review by amiclaus — APPROVED** [review]

File: projects/eval-adis1650x/src/examples/dummy/dummy_example.c (hunk 1 above)
Comment (CiprianRegus) [inline projects/eval-adis1650x/src/examples/dummy/dummy_example.c:113]: maybe cast instead?

**Review by CiprianRegus — APPROVED** [review]

**Review by buha — APPROVED** [review]
i will merge this with the mention that i'd really like that "dummy" renamed to something else at some point :D

**Pull request merged**